Makefile

```diff
@@ -179,6 +179,7 @@ destroy_remote:  ## Destroy the environment
 		fqdn=`jq -r '.tsb_fqdn' terraform.tfvars.json`; \
 		address=`jq -r "if .ingress_ip.value != \"\" then .ingress_ip.value else .ingress_hostname.value end" outputs/terraform_outputs/terraform-tsb-mp.json`; \
 		cd "tsb/fqdn/$$cloud"; \
+		terraform init; \
```

Comment:
Why is the `init` needed on a destroy phase?

Reply:
this bit me last week, when doing destroy it complained of a "missing register_fqdn module" or something similar. Adding this init fixed it.

Reply:
```
$ make destroy
╷
│ Error: Module not installed
│
│   on main.tf line 8:
│    8: module "register_fqdn" {
│
│ This module is not yet installed. Run "terraform init" to install all modules required by this configuration.
╵
make: *** [destroy_remote] Error 1
```